WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -118,5 +89,60 @@ protected override void ResetUi()
 			base.ResetUi();
 			PayjoinEndPoint = "";
 		}
+
+		private IPsbtSigner GetSigner()
+		{
+			return !Wallet.KeyManager.IsHardwareWallet
+				? null
+				: new HwiPsbtSigner(new HwiClient(Global.Network), b => IsHardwareBusy = b);
```

Comment:
Isn't the opposite easier to read?
```suggestion
			return Wallet.KeyManager.IsHardwareWallet
				? new HwiPsbtSigner(new HwiClient(Global.Network), b => IsHardwareBusy = b)
				: null;
```

Reply:
done